airgun/views/subscription.py

```diff
@@ -0,0 +1,102 @@
+from widgetastic.widget import (
+    Checkbox,
+    FileInput,
+    Text,
+    TextInput,
+    View,
+)
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin, SatTab
+from airgun.widgets import DeleteSubscriptionConfirmationDialog, SatTable, ProgressBar
+
+
+class SubscriptionListView(BaseLoggedInView, SearchableViewMixin):
+    """List of all subscriptions."""
+    table = SatTable(
+        locator='//*[@id="subscriptions-table"]//table',
+        column_widgets={
+            'Select all rows': Checkbox(locator=".//input[@type='checkbox']"),
```

Comment:
not sure that it work in that way:
`0: Checkbox(locator=".//input[@type='checkbox']"),`

Reply:
Sorry, missed that one.
Unfortunately, it won't work. If I use numbered index in column_widgets definition, Table widget will ignore my column content definition and try to find widget automatically. First column in table contains checkbox and (hidden) label. This will fool table widget into thinking that column contains text. Trying to change checkbox state will result in error:

```
Traceback (most recent call last):
  File "/home/mzalewsk/.local/share/virtualenvs/robottelo-ps2A3-Bl/lib/python3.6/site-packages/widgetastic/log.py", line 117, in wrapped
    result = f(self, *args, **kwargs)
  File "/home/mzalewsk/.local/share/virtualenvs/robottelo-ps2A3-Bl/lib/python3.6/site-packages/widgetastic/widget.py", line 41, in wrapped
    return method(self, Fillable.coerce(value), *args, **kwargs)
  File "/home/mzalewsk/.local/share/virtualenvs/robottelo-ps2A3-Bl/lib/python3.6/site-packages/widgetastic/widget.py", line 1309, in fill
    self.column_name or self.position, value, self.text))
TypeError: Cannot fill column Select all rows, no widget and the value differs (wanted to fill True but there is 'Select row'
```

I know that `Select all rows` is awkward, but it needs to stay this way.